src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -1787,63 +1663,22 @@ resource pipeline_ExecuteETL 'Microsoft.DataFactory/factories/pipelines@2018-06-
 
 //------------------------------------------------------------------------------
 // msexports_ETL_ingestion pipeline
-// Trigger: msexports_ExecuteETL
+// Triggered by msexports_ExecuteETL
 //------------------------------------------------------------------------------
 @description('Transforms CSV data to a standard schema and converts to Parquet.')
 resource pipeline_ToIngestion 'Microsoft.DataFactory/factories/pipelines@2018-06-01' = {
   name: '${safeExportContainerName}_ETL_${safeIngestionContainerName}'
   parent: dataFactory
   properties: {
     activities: [
-      // (start) -> Wait -> FolderArray -> Scope -> Metric -> Date -> File -> Folder -> Delete Target -> Convert CSV -> Delete CSV -> (end)
-      // Wait
-      {
-        name: 'Wait'
-        type: 'Wait'
-        dependsOn: []
-        userProperties: []
-        typeProperties: {
-          waitTimeInSeconds: 60
-        }
-      }
-      // Set FolderArray
-      {
-        name: 'Set FolderArray'
-        type: 'SetVariable'
-        dependsOn: [
-          {
-            activity: 'Wait'
-            dependencyConditions: [
-              'Completed'
-            ]
-          }
-        ]
-        userProperties: []
-        typeProperties: {
-          variableName: 'folderArray'
-          value: {
-            value: '@split(pipeline().parameters.folderName, \'/\')'
-            type: 'Expression'
-          }
-        }
-      }
-      // Set Scope
-      {
+            {
         name: 'Set Scope'
         type: 'SetVariable'
-        dependsOn: [
-          {
-            activity: 'Set FolderArray'
-            dependencyConditions: [
-              'Completed'
-            ]
-          }
-        ]
         userProperties: []
         typeProperties: {
           variableName: 'scope'
           value: {
-            value: '@replace(split(pipeline().parameters.folderName,variables(\'folderArray\')[sub(length(variables(\'folderArray\')), if(greater(length(variables(\'folderArray\')[sub(length(variables(\'folderArray\')), 2)]), 12), 3, 4))])[0],\'${exportContainerName}\',\'${ingestionContainerName}\')'
+            value: '@replace(split(pipeline().parameters.blobPath,split(pipeline().parameters.blobPath, \'/\')[sub(length(split(pipeline().parameters.blobPath, \'/\')), 4)])[0],\'${exportContainerName}\',\'${ingestionContainerName}\')'
```

Comment:
These functions are difficult to read. Should we create more variables to make it easier to understand?

Reply:
Yes.  I'll open a new PR for that.